core/src/test/java/org/apache/iceberg/view/ViewCatalogTests.java

```diff
@@ -63,6 +63,14 @@ public abstract class ViewCatalogTests<C extends ViewCatalog & SupportsNamespace
 
   @TempDir private Path tempDir;
 
+  protected String baseViewLocation(TableIdentifier identifier) {
+    String combinedPath = tempDir.toUri().toString();
+    combinedPath =
+        RewriteTablePathUtil.combinePaths(combinedPath, identifier.namespace().toString());
+    combinedPath = RewriteTablePathUtil.combinePaths(combinedPath, identifier.name());
```

Comment:
I'm not sure it's a good idea to use RewriteTablePathUtil here. I believe the default behavior should be to do what was previously done with the pathing, which subclasses can then override if needed. That's also what we do in the `CatalogTests`